### ISSUE TITLE
Building Damage Assessment ckpt download

### ISSUE THREAD

**Issue by kuaiqushangzixiba** [ChenHongruixuan/MambaCD#74]
can you upload Building Damage Assessment ckpt download ,please

**Comment by kuaiqushangzixiba** [comment]
Does the holdout data and tier3 data been used?

**Comment by kuaiqushangzixiba** [comment]
I noticed that the readme sets the path for the holdout data, but is it used in the code? It seems that the holdout data is not being used.

**Comment by ChenHongruixuan** [comment]
Hi @kuaiqushangzixiba ,

The holdout set is used. The form of our code is kept uniform across the three tasks. You can easily modify the current code to apply the holdout set.

BTW, we will upload the ckpt of MambaBDA soon. 

Best,

**Comment by heikeyuhuajia** [comment]
Hi, I noticed that the official new release of xview has four folders after unpacking, they are: train, test, holdout and tier3. my questions are:

Q1. did you use train for training, holdout as validation set to adjust the hyper-params, and test as the final test set? 

Q2. tier3 is not needed, right?

Also, I noticed that you said in issue #27: 

> Please try to not use lovasz softmax loss or enlarge the crop size. Or you can pre-crop the xBD dataset and exclude the samples whose label values are all 0.

However, in changemamba paper: 

> By utilizing the Lovasz-softmax loss function to solve the sample imbalance problem, 

it seems to use lovasz softmax. my question is: 

Q3. Is lovasz softmax required for BDA?

Looking forward to your reply, best wishes

**Comment by ChenHongruixuan** [comment]
Hi @kuaiqushangzixiba ,

thank you so much for your question. 

A1: yes. 

A2: tier3 is needed. you need to combine it with tier1 together. 

A3: The Lovasz-softmax loss function is used in ChangeMamba. The reason I told that user not to use the Lovasz-softmax loss is that he/she was experiencing problems caused by this loss function, even though we did not encounter this problem in our experiments.

Best,
